data-raw/DATASET.R

```diff
@@ -1,26 +1,27 @@
 # data.frame of colnames = c(psid, water_system_name, county, zipcode)
-water_systems <- tibble(psid = c("103039", "103040"),
-                        water_system_name = c("A","B"),
-                        county = c("D","E"),
-                        zipcode = c(90201, 94601))
+# water_systems <- tibble(psid = c("103039", "103040"),
+#                         water_system_name = c("A","B"),
+#                         county = c("D","E"),
+#                         zipcode = c(90201, 94601))
+water_systems <- readr::read_rds("data-raw/water_systems.rds")
 usethis::use_data(water_systems, overwrite=TRUE)
```

Comment:
Question: by changing this pkg-object and the two others in this script (analytes, psid_analyte) to the "real objects" will this break @ERGZ 's get_metadata.R tests?